### PR TITLE
<LabelRating />

### PR DESCRIPTION
**Pull request by dmytrozavgorodniy**
Please check functionality and implementation of component

labelRating_ios
labelRating_android

### CODE REVIEW

File: src/LabelRating/LabelRating.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
+import React, { Component, PropTypes } from 'react'
+import { View, Text, StyleSheet } from 'react-native'
+import StylePropType from '../utils/StylePropType'
+import ColorPallete from '../utils/ColorPallete'
+
+export default class LabelRating extends Component {
+  static propTypes = {
+    title: PropTypes.string.isRequired,
+    rating: PropTypes.string,
```
Comment (itsmepetrov) [inline src/LabelRating/LabelRating.js]: Set `rating` default props to `0`

File: src/LabelRating/LabelRating.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  render() {
+    const {
+      title,
+      rating,
+      style,
+      titleStyle,
+      ratingStyle,
+    } = this.props
+
+    const isRatingAvailable = rating && rating.length > 0
```
Comment (itsmepetrov) [inline src/LabelRating/LabelRating.js]: We should render `rating` always, even if `rating` equal to `0`

File: src/LabelRating/LabelRating.js (hunk 1 above)
Comment (isnifer) [inline src/LabelRating/LabelRating.js]: And it should be a number, not a string

Comment (dmytrozavgorodniy) [inline src/LabelRating/LabelRating.js]: Sure, I'll change it

File: src/LabelRating/LabelRating.js (hunk 2 above)
Comment (dmytrozavgorodniy) [inline src/LabelRating/LabelRating.js]: Sure

File: __tests__/e2e/04_LabelRating.test.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+  try {
+    await helper.openExampleFor('<LabelRating />')
+
+    for (const group of groups) {
+      const currentLabelRatingsParentId = group.labelRatingsParentId
+      const currentLabelRatings = group.labelRatings
+
+
+      for (const labelRating of currentLabelRatings) {
```
Comment (isnifer) [inline __tests__/e2e/04_LabelRating.test.js]:
```js
for (const [index, labelRating] of currentLabelRating.entries()) {
```

**Pull request merged**